www/js/diary/services.js

```diff
@@ -761,7 +761,7 @@ angular.module('emission.main.diary.services', ['emission.plugin.logger',
     }
 
     var confirmedPoints2Geojson = function(trip, locationList) {
-      var sectionCoordinates = locationList.map(function(point) {
+      var sectionCoordinates = locationList?.map(function(point) {
```

Comment:
Instead of all the checks to see if the location list is present and the user input is present, maybe we should continue to assume that they always exist and just ensure that the untracked time has a zero length list and an empty user input.

Reply:
According to the data model, `user_input` exists on confirmed objects but not cleaned objects.

So wouldn't it be strange to insert an empty `user_input` on the CLEANED_UNTRACKED type?

Reply:
Good point. We should leave the `user_input` out of `CLEANED_UNTRACKED` time.
We could still have a zero array for locations though...